custom_components/ha_washdata/frontend.py

```
@@ -63,15 +75,155 @@ class LovelaceResourceItem(TypedDict, total=False):
+        fd, tmp_name = tempfile.mkstemp(dir=str(path.parent), suffix=".gz.tmp")
+        os.close(fd)
+        tmp = Path(tmp_name)
+        try:
+            with open(path, "rb") as src, gzip.open(str(tmp), "wb", compresslevel=9) as dst:
+                shutil.copyfileobj(src, dst)
+            os.replace(tmp, gz)
+        except BaseException:
+            tmp.unlink(missing_ok=True)
+            raise
+        _LOGGER.debug("Wrote compressed asset %s", gz.name)
+    except Exception as exc:  # pylint: disable=broad-exception-caught
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)